sos/report/plugins/ceph.py

```diff
@@ -117,14 +124,17 @@ def setup(self):
             "time-sync-status",
         ]
 
-        self.add_cmd_output([
-            "ceph %s" % s for s in ceph_cmds
-        ])
+        ceph_osd_cmds = [
+            "ceph-volume lvm list",
+        ]
 
         self.add_cmd_output([
             "ceph %s --format json-pretty" % s for s in ceph_cmds
         ], subdir="json_output", tags="insights_ceph_health_detail")
 
+        for service in self.services:
+            self.add_journal(units=service)
+
```

Comment:
This is automatically handled already.

Reply:
This is why it was removed in my git diff ... I was wondering why. Fixed